core/src/mast/mod.rs

```diff
@@ -198,6 +196,10 @@ impl MastForest {
         Some(id_remappings)
     }
 
+    pub fn set_decorators(&mut self, node_id: MastNodeId, decorator_list: DecoratorList) {
```

Comment:
Created this to keep the code consistent. But this method should probably be replaced by 
```rust
mast_forest.get_node_by_id(node_id).set_decorators(decorator_list)
```

Reply:
I think we can replace it with an explicit match where we use this method, see here: https://github.com/0xPolygonMiden/miden-vm/pull/1531#discussion_r1847888167